substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/LLVMFeature.java

```diff
@@ -142,3 +155,27 @@ private static void checkLLVMVersion() {
         }
     }
 }
+
+@AutomaticFeature
+@Platforms(Platform.AMD64.class)
+class LLVMAMD64TargetSpecificFeature implements Feature {
```

Comment:
Can we add AArch64 as well?

Reply:
I was thinking about putting this in when we actually support ARM, instead of having bits of ARM support hanging around which might make it look like it works with ARM when it doesn't

Reply:
OK.